sdk/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
```
./tests.sh: line 33: ./build.sh: Permission denied
```
IMO it will be also better to add:
```
chmod +x tests.sh build.sh
```
possibility redundant but works on new repo